fiboa_cli/datasets/commons/euro_land.py

```diff
@@ -0,0 +1,50 @@
+class EuroLandConverterMixin:
+    """
+    Datasets have been published by the
+    [Euroland project](https://europe-land.eu/news/harmonized-database-of-european-land-use-data-published/)
+    as open data. See https://zenodo.org/records/14384070 for a list of open data sets.
+
+    Use this abstract class to create converters based on the euroland repository
+    Subclasses should declare: id, short_name, title, description, providers, license, attribution,
```

Comment:
I find the interface as such a bit weak. If it's only clear from a comment what's needed for an implementation, it's hard to understand IMHO. An interface should be something in code logic, a method or abstract class or so...